packages/react-core/.eslintrc.js

```diff
@@ -0,0 +1,72 @@
+module.exports = {
+  env: { node: true },
```

Comment:
QQ, why is `node` env set to `true` here?

Reply:
It enables `node` global variables, more info [here](https://eslint.org/docs/user-guide/configuring/language-options#specifying-environments)